com.github.Qv2ray.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "com.github.Qv2ray",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "qv2ray",
+    "rename-desktop-file": "qv2ray.desktop",
+    "rename-icon": "qv2ray",
+    "rename-appdata-file": "qv2ray.metainfo.xml",
+    "cleanup": [
+        "/include",
+        "/lib/debug",
+        "*.la",
+        "*.a"
+    ],
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--talk-name=com.canonical.AppMenu.Registrar",
```

Comment:
```suggestion
```
You already own this just by using KDE runtime 5.13 see https://github.com/KDE/flatpak-kde-runtime/blob/qt5.13/org.kde.Sdk.json.in#L65